doc/adr/0002-summary-of-decisions-thus-far.md

```diff
@@ -0,0 +1,85 @@
+# 2. Summary of decisions thus far
+
+Date: 2024-10-09
+
+## Status
+
+Accepted
+
+## Context
+
+This is a high-level summary of previous decisions that have been made on this
+project. ADRs will be used for all new architectural decisions.
+
+## Decision
+
+Below is a list with the specific explanations on the architectural decisions
+that have been made thus far in this project.
+
+#### Use Django and Django Admin.
+
+Previously, a Web 1.0 Python application was being built to create this
```

Comment:
I don't think other devs will understand what "Web 1.0 application" means without more context. You could say we were using Flask, and we were building our own custom HTML templates. Then we decided that Django's admin panel had most of what we needed in terms of UI.

Reply:
Made this change!